server/eclipse-project/src/main/java/us/freeandfair/corla/model/CountyDashboard.java

```diff
@@ -887,7 +887,27 @@ public Integer auditedSampleCount() {
   public void setAuditedSampleCount(final int the_audited_sample_count) {
     my_audited_sample_count = the_audited_sample_count;
   }
-
+  
+  /** 
+   * Aborts all audits in the county. This changes the state of any audits
+   * that have not achieved their risk limit to ABORTED.
+   */
+  public void endAudits() {
+    for (final CountyContestComparisonAudit ccca : my_comparison_audits) {
+      ccca.endAudit();
+    }
+  }
+  
+  /**
+   * Updates the status for all audits in the county. This changes their states
+   * based on whether they have achieved their risk limits.
+   */
+  public void updateAuditStatus() {
+    for (final CountyContestComparisonAudit ccca : my_comparison_audits) {
+      ccca.updateAuditStatus();
```

Comment:
If one of these calls failed, would all the transactions roll back, or not? Same question applies above.

Reply:
The entire transaction would roll back. However, I don't see any realistic way that those calls can fail while simultaneously having the actions that preceded them succeed.

Reply:
Great.